templates/Layout/DNRoot_environment.ss

```diff
@@ -37,49 +38,64 @@
 					$DeploymentMessages
 				<% end_if %>
 
-				<h4>Environment Details:</h4>
-
-				<ul>
+				<h3>Currently deployed  
 					<%-- Display Environment URL --%>
 					<% if $URL %>
-						<li>
-							<span class="fa fa-link"></span>
-							<a href="$URL">View site at $BareURL</a>
-						</li>
+						<small><a class="text-muted" href="$URL">$URL</a></small>
 					<% end_if %>
+				</h3>
 
-					<%-- Display logs link for environment --%>
-					<% if $LogsLink %>
-						<li>
-							Logs: <a href="$LogsLink">View Logs for $Name</a>
-						</li>
-					<% end_if %>
+				<div class="row text-lg">
+					<div class="col-md-6">
+						<ul>
 
-					<%-- Display metrics for environment --%>
-					<% if $Project.HasMetrics %>
-						<li>
-							Metrics:
-							<a href="naut/project/$Project.Name/environment/$Name/metrics">
-								See graphs for this environment
-							</a>
-						</li>
-					<% end_if %>
+							<% if $CurrentBuild %>
+								<li>
+									Latest change: $CurrentBuild.Message
+								</li>
+							<% end_if %>
 
-					<%-- Display current build on environment --%>
-					<% if $CurrentBuild %>
-						<li>
-							Deployed Revision:
-							<% with $CurrentBuild %><% include GitBuildReference %><% end_with %>
-						</li>
-						<li>
-							<a href="$CurrentBuild.Link">View deployment log</a>
-						</li>
-					<% else %>
-						<li>
-							No deployments have been made
-						</li>
-					<% end_if %>
-				</ul>
+							<%-- Display current build on environment --%>
+							<li>
```

Comment:
is the only difference <% include GitBuildReference %> so far that is the only thing missing right?